android/src/toga_android/widgets/detailedlist.py

```diff
@@ -4,10 +4,39 @@
 from android.app import AlertDialog
 from android.content import DialogInterface
 from android.graphics import Color, Rect
+from android.os import Build
+from android.util import TypedValue
 from android.view import Gravity, View
 from android.widget import ImageView, LinearLayout, RelativeLayout, ScrollView, TextView
 from java import dynamic_proxy
 
+from .base import Widget
+
+
+def _resolve_theme_color(view, attr_id, fallback):
```

Comment:
These theme attributes are guaranteed to be in every theme, so there's no need for the fallback mechanism. If we somehow don't find them, we should just throw an exception.

There's probably also no need to support a ColorStateList, unless you have reason to believe otherwise.